packages/mdc-floating-label/README.md

```diff
@@ -69,14 +59,20 @@ Mixin | Description
 `mdc-floating-label-shake-animation($modifier)` | Applies shake keyframe animation to label.
 `mdc-floating-label-float-position($positionY, $positionX, $scale)` | Sets position of label when floating.
 
-### `MDCFloatingLabel`
+## `MDCFloatingLabel` Properties and Methods
+
+### Methods
 
 Method Signature | Description
 --- | ---
 `shake(shouldShake: boolean) => void` | Shakes or stops shaking the label, depending on the value of `shouldShake`. Proxies to the foundation method of the same name.
 `float(shouldFloat: boolean) => void` | Floats or docks the label, depending on the value of `shouldFloat`. Proxies to the foundation method of the same name.
 `getWidth() => number` | Returns the width of the label element.
 
+## Usage Within Frameworks
+
+If you are using a JavaScript framework, such as React or Angular, you can create a `MDCTextField` for your framework. Depending on your needs, you can use the _Simple Approach: Wrapping MDC Web Vanilla Components_, or the _Advanced Approach: Using Foundations and Adapters_. Please follow the instructions [here](../../../docs/integrating-into-frameworks.md).
```

Comment:
`MDCTextField` should be Floating label (copypaste, and <COMPONENT_NAME> doesn't refer to the literal JS name, that's <JS_COMPONENT_NAME>)